--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -266,9 +266,9 @@
 ###############################################################################
 # Blueprint Maven
 ###############################################################################
-bazel_dep(name = "rules_jvm_external", version = "6.10")
+bazel_dep(name = "rules_jvm_external", version = "6.10", dev_dependency = True)
 
-maven = use_extension("@rules_jvm_external//:extensions.bzl", "maven")
+maven = use_extension("@rules_jvm_external//:extensions.bzl", "maven", dev_dependency = True)
 maven.install(
     name = "blueprint_maven_dependencies",
     artifacts = [
